# This message serves as switch command for the EPS motor power
Header header
bool MotorPower 					# Motor power switch

